src/service/chat.js

```diff
@@ -161,9 +161,9 @@ const getOrCreateAnonymousChannel = async (userId) => {
   }
 };
 
-const moveChatToSigned = async ({oldChannelId, targetUserId}) => {
+const moveChatToSigned = async ({oldChannelId, targetUserId, source}) => {
   try {
-    const response = await api.post('/chat/move-to-sign', {oldChannelId, targetUserId});
+    const response = await api.post('/chat/move-to-sign', {oldChannelId, targetUserId, source});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `source` parameter has been added to the `moveChatToSigned` function and is included in the payload sent to the API endpoint. However, there's no error handling for the case when `source` is not provided or is null. This could lead to unexpected behavior or errors from the API. Consider adding a check for `source` before making the API call.
```diff
+    if (!source) {
+      throw new Error('Source must be provided');
+    }
     const response = await api.post('/chat/move-to-sign', {oldChannelId, targetUserId, source});
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->